### ISSUE TITLE
Проблемы и баги

### ISSUE THREAD

**Issue by ghost** [anijackich/sdamgia-api#1]
Здравствуйте. Во время работы с Вашей библиотекой я нашёл несколько проблем:

1. После каждого обращения, в консоль выводится информация про какую-то задачу (а точнее "Математика, 505452").
2. Текст решения представляется в какой-то странной форме: `... Решение.Андрей выучил 60\xa0–\xa03\xa0=\xa057 вопросов. Поэтому вероятность того, ...`
3. Также, я не совсем понимаю, как работает метод `get_problem_by_id`. Задание по информатике №10310 библиотека не видит/или возвращает неправильный ответ и само задание.

**Comment by anijackich** [comment]
Здравствуйте, спасибо за вопрос!

Были небольшие проблемы с кодировкой, это будет исправлено со следующей версии

Что насчет 3, СдамГИА использует защиту от ddos-атак от CloudFlare. Поэтому иногда бывает, что запросы на сайт блокируются и, как следствие, информацию о задаче получить невозможно. Я сейчас работаю над этим.

**Comment by ghost** [comment]
Благодарю за оперативный ответ, буду ждать новую версию. Спасибо Вам большое за прекрасную библиотеку. Удачи!

**Comment by ghost** [comment]
Кстати, хотелось-бы ещё добавить поддержку как Сдам ОГЭ, так и Сдам ЕГЭ.

**Comment by anijackich** [comment]
Да, это хорошая идея, это так же будет добавлено с новой версии
